content/contribute.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Contributing to gRPC
+---
+
+To contribute to gRPC documentation, please fork the [gRPC repository on GitHub](https://github.com/grpc/grpc.io) and start submitting pull requests.
```

Comment:
Nit: drop "please"

Reply:
I’ve opted to leave all of the actual content as-is. We can do a pass at content in other PRs.